06-building-a-program-start-to-finish/writing-tests/output/spec.js

```diff
@@ -0,0 +1,122 @@
+/* My Test Setups */
+/* global describe it expect */
+import myBook from './books.json';
+import emptyBook from './emptyBook.json';
+import invalidKeys from './Invalidkeys.json';
+
+/* My Suites */
+describe('Class and Method Instantaion', () => {
+  const myClass = new InvertedIndex();
```

Comment:
It is advised that when you create an instance of a class it should exactly match the name.